### ISSUE TITLE
web scripts and talkback cause crashes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#175]
```
What steps will reproduce the problem?
1. Have talkback, web scripts running and open netflix
2. While talkback is running attempt to swipe or explore screen, currently you 
would be at the log-in screen.
3. As your exploring, talkback does not say any thing, turn talkback off and 
the web scripts will talk while moving your finger around the screen.

What is the expected output? What do you see instead?
I am attempting to log-in with my netflix email but talkback is not reading the 
screen.  What I see with talkback is a loading, then blank screen, I know for a 
fact there is edit boxes I need to type my logp-in information in but talkback 
can not focus to the window where the log-in fields are located.

What version of the product are you using? On what operating system?
I am using the latest netflix, runing jelly bean 4.1.1 on a US t-mobile branded 
Galaxy S3.

Please provide any additional information below.


```

Original issue reported on code.google.com by `danny.ma...@gmail.com` on 7 Feb 2013 at 9:46
- Merged into: #160

**Comment by GoogleCodeExporter** [comment]
```
TalkBack is actually still active.  Netflix is muting and locking the media 
volume stream, which cannot be overridden by TalkBack.
```

Original comment by `caseybur...@google.com` on 12 Feb 2013 at 3:23
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
so what can be done about this issue?  Can talkback be given another
media streem when netflix locks the main media streem?  I can log-in
with my wife's help with talkback off and then everything works.  When
you log-in that is the issue, no speach feedback what so ever.  I
could try braille if you want me to see if can log-in that away.
```

Original comment by `danny.ma...@gmail.com` on 12 Feb 2013 at 3:16

**Comment by GoogleCodeExporter** [comment]
```
I have spoken to the Netflix Technical department, got a hold of someone who 
contacted netflix's moble dev team and was told they have been working with 
your talkback team to solve this media stream issue.  If that is not the case, 
then what as a consumer can I do to get this issue resolved?  I am willing to 
beta test netflix's app if it takes just that to fix this issue.
```

Original comment by `danny.ma...@gmail.com` on 13 Feb 2013 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
Danny,
Just surveyed the entire team.  We've not heard from Netflix regarding this.  
If you can happen to get back in touch with them, feel free to give them my 
email address caseyburkhardt@google.com and we can help them find a reasonable 
solution to this issue.

Thanks
```

Original comment by `caseybur...@google.com` on 20 Feb 2013 at 12:57
